lib/t/push_notifications/dispatch_job.ex

```diff
@@ -126,6 +126,16 @@ defmodule T.PushNotifications.DispatchJob do
     |> Repo.one()
   end
 
+  defp handle_type("deactivated" = type, args) do
+    %{"user_id" => user_id} = args
```

Comment:
see the warning above

Reply:
yeah, saw it but couldn't understand since it point at L14:
defp handle_type("match", args) do
    %{"match_id" => match_id} = args

however we also have similar guys like:
1) defp handle_type("timeslot_cancelled" = type, args) do
    %{"match_id" => match_id} = args
or
2) defp handle_type("timeslot_ended", args) do
    %{"match_id" => match_id} = args

which also seem to have the same name and arity (number of arguments), but do not produce this kind of warning

what am I missing here?

Reply:
> clauses with the same name and arity (number of arguments) should be grouped together

you need to put the new clause next to the other ones, above `profile_name/1`